packages/terra-consumer-nav/CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 ChangeLog
 =========
 
+# 0.1.0-BETA.3 - (--)
```

Comment:
yeah that's my bad, i thought the release date was supposed to go here since releases may not always be the same day as the commit. I'll update this